src/log/LogManager.cpp

```diff
@@ -61,15 +64,15 @@ void DLogManager::registerFileAppender() {
 /*!
   \brief Return the path file log storage.
 
-  \brief DLogManager::getlogFilePath 获取日志文件路径
-  \brief 默认日志路径是 ~/.cache/organizationName/applicationName.log
-  \brief 如果获取 HOME 环境变量失败将不写日志
+  \brief DLogManager::getlogFilePath Get the log file path
+  \brief The default log path is ~/.cache/OrganizationName/ApplicationName.log
+  \brief If you fail to get the $HOME environment variables, you will not write a log
   \sa registerFileAppender
  */
 QString DLogManager::getlogFilePath()
 {
-    // 不再构造时去设置默认logpath(且mkdir), 而在getlogPath时再去判断是否设置默认值
-    // 修复设置了日志路径还是会在默认的位置创建目录的问题
+    // No longer set the default logpath when constructing(and mkdir), And   when getlogPath to determine whether to set the default value
+    // The repair setting the log path or the problem of creating a directory in the default position
     if (DLogManager::instance()->m_logPath.isEmpty()) {
```

Comment:
No longer set the default log path (and mkdir) when constructing now, instead set the default value if it's empty when getlogFilePath is called.
Fix the problem that the log file is still created in the default path when the log path is set manually.